blue_modules/notifications.js

```
@@ -265,89 +245,63 @@ const _setPushToken = async token => {
+      .then(({ status }) => {
+        if (status === 'granted') {
+          PushNotification.configure({
+            onRegister: async token => {
+              console.debug('TOKEN:', token);
+              alreadyConfigured = true;
+              await _setPushToken(token);
+              resolve(true);
+            },
+            onNotification: async notification => {
+              const payload = { ...notification, ...notification.data };
+              if (notification.data && notification.data.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)